### ISSUE TITLE
Ontology file (.OBO) fails to load

### ISSUE THREAD

**Issue by taytayp** [tripal/tripal#184]
Upon loading a .obo ontology file to the Chado OBO loader, and running the job through drush, the error returned is:

`Invalid OBO ID provided: '0'.`
`WD tripal_job: Invalid OBO ID provided: '0'.`

The file in question does not have any IDs similar to '0.'

[ftp://ftp.ebi.ac.uk/pub/databases/chebi/ontology/chebi.obo](url) is the file in question.

**Comment by bradfordcondon** [comment]
Hi there,

Sorry you haven't gotten any feedback yet.  
Are you still getting this issue?  I've had it before but when I re-run the import it has worked- I wonder if creating the new ontology reference needs to happen before the job runs?

**Comment by taytayp** [comment]
No worries about the delay.

Still seems to be an issue. I am aware it works if you select the ontology again upon revisiting a page, and am currently uploading one now.

**Comment by shawnawsu** [comment]
Hi @taytayp ,

I just pushed an update to the 7.x-3.x branch that should fix this problem. 

Let me know if it's working for you and I can close this issue. 

Thanks!
Shawna

**Comment by shawnawsu** [comment]
HI @taytayp ,

I am going to close this ticket. If the loader is still giving you issues with the newest 7.x-3.x branch please let us know in a new ticket. 

Thanks!
Shawna
